ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts

```diff
@@ -0,0 +1,120 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+import {
+    cancelAllCveExceptions,
+    fillAndSubmitExceptionForm,
+    selectSingleCveForException,
+    verifyExceptionConfirmationDetails,
+    verifySelectedCvesInModal,
+    visitWorkloadCveOverview,
+} from '../workloadCves/WorkloadCves.helpers';
+import { selectors as workloadCVESelectors } from '../workloadCves/WorkloadCves.selectors';
+import { selectors as vulnSelectors } from '../vulnerabilities.selectors';
+import { visit } from '../../../helpers/visit';
+import { pendingRequestsPath } from './ExceptionManagement.helpers';
+
+const deferralComment = 'Defer me';
+const deferralExpiry = 'When all CVEs are fixable';
+const deferralScope = 'All images';
+
+describe('Exception Management Request Details Page', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
```

Comment:
Would it be would making this 2nd beforeEach block just an else, or fall-through block of a single beforeEach block?

Reply:
Ah, you're suggesting merging the two `beforeEach` into one. That would be reasonable to do. I see an issue where the tests are not properly canceling the exceptions. Maybe there's a race condition involved with these two `beforeEach` functions. Merging them might resolve that issue as well.